tests/jballerina-unit-test/src/test/resources/test-src/workers/worker-in-lock.bal

```diff
@@ -0,0 +1,9 @@
+function testWorkerInsideLock() {
+    lock {
+        fork {
```

Comment:
Can we add a test for start too?

Reply:
Will add

Reply:
Also Add test case for nested fork statements.

Reply:
+1